package.json

```diff
@@ -26,7 +26,7 @@
         "slackchannel": "#billing-errors",
         "resources": {}
       },
-      "pmapi": {
+      "pmApi": {
```

Comment:
It just occurred to me: you need to change line 20 of `.github/workflows/monitor.yml`.

I think we should have separate jobs in the monitor.yml file for each goalie, that way the goalie can drill down to the specific job rather than skimming through the collected tests.

Reply:
I think that might be best to do in another PR (I can make the change right after this is merged).